finalPrep/modZeroHero.js

```diff
@@ -48,24 +77,81 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 //assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
+function assessSituation2 (dangerLevel, announcement, excuse) {
+  if (dangerLevel >= 50){
+    console.log(excuse);
+  }
+  else if (dangerLevel < 10){
+    console.log('Meh. Hard pass.');
+  }
+  else {
+    console.log(announcement);
+  }
+}
+assessSituation2 (99, `${announcement}`, `${excuse}`);
+assessSituation2 (21, `${announcement}`, `${excuse}`);
+assessSituation2 (3, `${announcement}`, `${excuse}`);
+
+
+
+
 // Declare a new variable - scaryMonster - assigned to an Object with the following key/values
 //   - name (string)
 //   - smell (string)
 //   - weight (integer)
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var address = {
+    number: 4320,
+    street: 'Walnut',
+    state: 'NM',
+    zip: 32094
+}
+
+var scaryMonster = {
```

Comment:
Really nice job with this! Cool to see the way you pulled the object literal out into its own variable 👍